src/app/component/ubs/components/ubs-submit-order/ubs-submit-order.component.scss

```diff
@@ -1,144 +1,55 @@
 .container {
   color: var(--ubs-primary-grey);
-  box-shadow: 1px 4px 8px rgb(100 114 125 / 18%);
   padding: 10px 25px 40px;
 }
 
-.header,
-.main {
-  width: 100%;
-  margin-top: 2rem;
-  margin-bottom: 3rem;
-}
-
-h3 {
-  font-weight: bold;
-}
-
-table {
-  width: 100%;
-  margin-top: 1rem;
-}
+.your-order {
+  background-color: #f8f9fb;
 
-tbody {
-  padding: 4rem 0;
-  border-bottom: 1px solid var(--ubs-primary-light-grey);
-  border-top: 1px solid var(--ubs-primary-grey);
-}
-
-.firstRow {
-  padding-top: 2rem;
-}
-
-.lastRow {
-  padding-bottom: 2rem;
-}
-
-th,
-td {
-  padding: 0.8rem 1rem 0.8rem 0;
-}
-
-th,
-label {
-  color: var(--ubs-secondary-grey);
-}
-
-.showTotal {
-  width: 100%;
-  display: flex;
-  justify-content: flex-end;
-  margin-top: 1rem;
-}
-
-.totalInfo {
-  text-align: right;
-  width: 25%;
-
-  p {
-    display: flex;
-    justify-content: space-between;
-    color: var(--ubs-secondary-grey);
-    padding-right: 0.5rem;
+  table {
+    width: 100%;
   }
 
-  span {
-    display: block;
+  th,
+  td {
+    padding: 0.8rem 1rem 0.8rem 0;
   }
 
-  small {
-    color: var(--ubs-primary-green);
+  .right-column {
+    width: 100px;
   }
 }
 
-.header {
-  margin-top: 3rem;
-}
-
-.order {
-  display: flex;
-  width: 100%;
-  justify-content: space-between;
-  padding-top: 2rem;
-}
-
-.finalCost {
-  text-align: end;
-}
-
-.personalData {
-  width: 60%;
-  margin-top: 5rem;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-}
-
-.data {
-  width: 100%;
-}
-
-.data:nth-child(2) {
-  margin-top: 3rem;
-}
-
-ul {
-  list-style: none;
-  padding: 0;
-}
-
-.bottom-select {
-  padding: 0;
-  margin-bottom: 2rem;
-}
-
-select,
-button {
-  height: 3rem;
-  border-radius: 0;
-}
+.order-details {
+  margin-top: 2rem;
+  background-color: #f8f9fb;
```

Comment:
Same color here

Reply:
The variable for color has been added with this commit 063e126